api/__tests__/app.test.js

```diff
@@ -222,6 +222,106 @@ describe("/api", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    it("should increment the specified article id's and add 1 vote and return the updated article", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({
+          inc_votes: 1,
+        })
+        .expect(200)
+        .then((res) => {
+          const { article } = res.body;
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
Expect an article_id that matches the id used in the parametric endpoint. Applies to other tests too